README.md

```diff
@@ -104,58 +114,120 @@ Contains Non-ASCII Characters: 12
+  secretfile            Location of de-identification secret file
 
 optional arguments:
-  -h, --help       show this help message and exit
+  -h, --help            show this help message and exit
+  -o OUTPUTFILE, --output OUTPUTFILE
+                        Specify an output file. Default is output/garbled.zip
 ```
 
-`garble.py` will package up the garbled PII files into a [zip file](https://en.wikipedia.org/wiki/Zip_(file_format)) called `garbled.zip`.
+`garble.py` will package up the garbled PII files into a [zip file](https://en.wikipedia.org/wiki/Zip_(file_format)) called `garbled.zip` and place it in the `output/` folder by default, you can change this with an `--output` flag if desired.
 
 Example execution of `garble.py` is shown below:
 
 ```
-$ python garble.py --source /Users/andrewg/projecs/anonlink-multiparty/data/siblings/system-a.csv --schema /Users/andrewg/Desktop/schema
-Salt value:
-generating CLKs:  21%|▏| 52.0/252 [00:00<00:00, 321clk/s, mean=1.16e+3, std=21.2generating CLKs: 100%|█| 252/252 [00:00<00:00, 420clk/s, mean=1.17e+3, std=36.1]
+$ python garble.py temp-data/pii.csv example-schema ../deidentification_secret.txt
 CLK data written to output/name-sex-dob-phone.json
-generating CLKs:  21%|▏| 52.0/252 [00:00<00:00, 417clk/s, mean=1.17e+3, std=22.4generating CLKs: 100%|███| 252/252 [00:00<00:00, 535clk/s, mean=1.17e+3, std=32]
 CLK data written to output/name-sex-dob-zip.json
-generating CLKs:  21%|▏| 52.0/252 [00:00<00:00, 340clk/s, mean=1.53e+3, std=73.1generating CLKs: 100%|█| 252/252 [00:00<00:00, 424clk/s, mean=1.54e+3, std=68.2]
 CLK data written to output/name-sex-dob-parents.json
-generating CLKs:  21%|▏| 52.0/252 [00:00<00:00, 423clk/s, mean=1.17e+3, std=23.7generating CLKs: 100%|███| 252/252 [00:00<00:00, 532clk/s, mean=1.17e+3, std=34]
 CLK data written to output/name-sex-dob-addr.json
+Zip file created at: output/garbled.zip
+```
+### [Optional] Household Extract and Garble
+
+You may now run `households.py` with the same arguments as the `garble.py` script, with the only difference being specifying a specific schema file instead of a schema directory - if no schema is specified it will default to the `example-schema/household-schema/fn-phone-addr-zip.json` (use `-h` flag for more information). NOTE: If you want to generate the testing and tuning files for development on a synthetic dataset, you need to specify the `-t` or `--testrun` flags
+
+The households script will do the following:
+  1. Attempt to group individuals into households and store those records in a csv file in temp-data
+  1. Create a mapping file to be sent to the linkage agent, along with a zip file of household specific garbled information.
+
+This information must be provided to the linkage agent if you would like to get a household linkages table as well.
+
+Example run:
+```
+$ python households.py temp-data/pii.csv ../deidentification_secret.txt
+Grouping individuals into households: 819it [01:04, 12.77it/s]
```

Comment:
"`819it`"? Maybe something got ... garbled, here? (sorry)

Would be good if this read something like:

```Grouping individuals into households: 100%|███████████████████████| 819/819 [01:12<00:00, 11.37it/s]```

Reply:
Ah yeah, i fixed that in the latest commit but forgot to update the readme. the enumerate() inside of tqdm doesn't give the bar but gives like an iteration count format